packages/bus-rdf-join/lib/ActorRdfJoin.ts

```diff
@@ -251,7 +251,11 @@ TS
             value: acc.value * (optional ? Math.max(1, cardinalityThis.value) : cardinalityThis.value),
           };
         }, { type: 'exact', value: 1 });
-      cardinalityJoined.value *= (await this.mediatorJoinSelectivity.mediate({ entries, context })).selectivity;
+      if (cardinalityJoined.value > 1e-9) {
```

Comment:
Could you add a comment here describing why this is needed?
The reason for this may get lost in the future.